applications/dashboard/controllers/api/LocalesApiController.php

```diff
@@ -71,9 +71,12 @@ private function getEnabledLocales(): array {
         $hasEnLocale = false;
         $result = [];
         foreach ($locales as $localeID => $locale) {
+            $localKey = (strlen($locale['Locale']) === 5) ?
+                substr($locale['Locale'], 0, 5) :
+                substr($locale['Locale'], 0, 2);
```

Comment:
I think we can just go with `$locale['Locale']`, in this case localeKey and regionalKey will both be the same, but it's ok.

We already do normalization of these by this point.

Reply:
i've set it to $locale['Locale']